deep_speech_2/evaluate.py

```diff
@@ -153,10 +169,12 @@ def evaluate():
             for _, transcript in infer_data
         ]
         for target, result in zip(target_transcripts, result_transcripts):
-            wer_sum += wer(target, result)
+            error_sum += error_rate_func(target, result)
             num_ins += 1
-        print("WER (%d/?) = %f" % (num_ins, wer_sum / num_ins))
-    print("Final WER (%d/%d) = %f" % (num_ins, num_ins, wer_sum / num_ins))
+        print("%s (%d/?) = %f" % \
```

Comment:
Remove `error_rate_info` and use `print("Error rate [%s] ....." % args.error_rate_type)` for simplicity ?